integration/oasis/stationgraph/node.go

```diff
@@ -0,0 +1,45 @@
+package stationgraph
+
+import (
+	"math"
+
+	"github.com/golang/glog"
+)
+
+type chargeInfo struct {
+	arrivalEnergy float64
+	chargeTime    float64
+	chargeEnergy  float64
+}
+
+type node struct {
+	id    nodeID
+	edges []*edge
+	chargeInfo
+}
+
+type nodeID int32
```

Comment:
`uint32`?

Reply:
I defined invalid NodeID as `const invalidNodeID = math.MinInt32`, so here use int32

I think I could also use uint32 and use maxuint32 as invalid then change to uint32